doc/classes/Basis.xml

```diff
@@ -64,20 +64,21 @@
 			<return type="Vector3">
 			</return>
 			<description>
-				Assuming that the matrix is a proper rotation matrix (orthonormal matrix with determinant +1), return Euler angles (in the YXZ convention: first Z, then X, and Y last). Returned vector contains the rotation angles in the format (X angle, Y angle, Z angle).
+				Returns the basis' rotation in the form of Euler angles (in the YXZ convention: first Z, then X, and Y last). The returned vector contains the rotation angles in the format (X angle, Y angle, Z angle). See [method get_rotation_quat] if you need a quaternion instead.
```

Comment:
I think `basis'` should be `basis's` since the ending apostrople is used for plurals with ownership.

Reply:
I didn't know about that rule, but it seems correct: https://www.thepunctuationguide.com/apostrophe.html#possessives

(And lol at English -- an exception for words only when followed by "sake"? Come on now.)